kedro_airflow_k8s/airflow_dag_template.j2

```diff
@@ -20,6 +22,8 @@ with DAG(
     params={"example_key": "example_value"},
 ) as dag:
 
+    pvc_name = '{{ project_name | safe | slugify }}.{{ execution_marker }}'
```

Comment:
Consider using `{{ ts_nodash }}` (https://airflow.apache.org/docs/apache-airflow/stable/macros-ref.html) -> it needs to be set on **execution** (to be random for different runs rather during pipeline generation (that may be done only once)

Reply:
Good catch, and that is what I was looking for :) Is it just sufficient to put it in place and Airflow will replace {{ ts_nodash }} during the execution - that's what you meant, right?

Reply:
Indeed. If I'm not mistaken, all the parameters pushed to PythonOperator are templated, so it should work OOTB